src/borg/manifest.py

```diff
@@ -166,6 +166,10 @@ def list_considering(self, args):
             match=args.match_archives,
             first=args.first,
             last=args.last,
+            older=args.older,
+            newer=args.newer,
+            oldest=args.oldest,
+            newest=args.newest,
```

Comment:
did you check all callers of `list_considering` whether they set up these attributes of `args` (== whether they support the corresponding cmd line options in their arg parser)?

Reply:
(if the option parser for these options is not added, you'ld get an `AttributeError` here)

Reply:
maybe a less fragile way would be to use `x=getattr(args, "x", correctdefault)`.

i also missed to do that for first and last.

Reply:
Yeah, that seems like a better way to do it.